1001-2000/LIT1824Mafteh.xml

```diff
@@ -77,6 +78,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <relation name="ecrm:CLP46i_may_form_part_of" active="LIT5139MagicPrayer" passive="LIT1824Mafteh"></relation>
 
          </listRelation>
+         <div type="edition">
+            <note>The following text part description is based on the manuscript <ref type="mss" corresp="MEAG002"/>which is incomplete.</note>
+            <div type="textpart" subtype="chapter" xml:id="Monday">
+               <label>Prayer for Monday</label>
```

Comment:
1) If you want to add a structure or text to a work record, you should take a full text witness (here, it  stops on Tuesday) in a way representative for the tradition. 
2) I have some doubts as to if the text is really structured according to the days of the week. I have never seen such a Meftahe seray. (if the means for this are notes in the upper margin, they do not necessarily reflect the structure)
3) Unless it is not done in one or another way, I suggest to add to LIT1824Mafteh the status of "general record". The collections with this title are of unstable composition, not much studied (possibly an issue can be done to discuss this).